plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/jdbc/TestingIcebergJdbcServer.java

```diff
@@ -64,12 +64,10 @@ public void execute(@Language("SQL") String sql)
     public String getJdbcUrl()
     {
         return format(
-                "jdbc:postgresql://%s:%s/%s?user=%s&password=%s&currentSchema=tpch,public",
```

Comment:
Did you mean to remove the `currentSchema` property?

Reply:
Yes, exactly.